.settings.xml

```diff
@@ -23,23 +23,15 @@
 				<repository>
 					<id>spring-snapshots</id>
 					<name>Spring Snapshots</name>
-					<url>https://repo.spring.io/libs-snapshot-local</url>
+					<url>https://repo.spring.io/snapshot</url>
 					<snapshots>
 						<enabled>true</enabled>
 					</snapshots>
 				</repository>
 				<repository>
 					<id>spring-milestones</id>
 					<name>Spring Milestones</name>
-					<url>https://repo.spring.io/libs-milestone-local</url>
-					<snapshots>
-						<enabled>false</enabled>
-					</snapshots>
-				</repository>
-				<repository>
-					<id>spring-releases</id>
-					<name>Spring Releases</name>
-					<url>https://repo.spring.io/release</url>
+					<url>https://repo.spring.io/milestone</url>
```

Comment:
Unlike `libs-milestone-local` that contains 3rd party deps, when we switch to `milestone` we have to get our 3rd party deps from somewhere... Maven Central. AFAIK we pretty much need to add MC everywhere that we do this replacement. That is my understanding. Is that correct though?